pallets/thea-council/src/lib.rs

```diff
@@ -68,9 +69,14 @@ pub mod pallet {
 	pub trait Config: frame_system::Config + xcm_helper::Config {
 		/// Because this pallet emits events, it depends on the runtime's definition of an event.
 		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
+		/// Unix timestamp provider
+		type TimeProvider: Time;
```

Comment:
@35359595 Time should not be used, it's not trusted, please use blocks as a notion of time.

Reply:
done